grails-app/controllers/org/pih/warehouse/api/SelectOptionsApiController.groovy

```diff
@@ -69,7 +69,13 @@ class SelectOptionsApiController {
     }
 
     def usersOptions = {
-        List<User> users = userService.findUsers(params)
+        params.roleTypes = params.list("roleTypes")
+        if (params.roleTypes && !params.location) {
+            params.location = session.warehouse?.id
+        }
+        List<User> users = userService.findUsers(params).collect {
+            [id: it.id, label: it.name, value: it.id ]
```

Comment:
maybe not for now, but maybe we should think about creating a dto called `SelectOptionDto` for those select options?
In all places we return the same object which is object with `id`, `label`, `value`.
It'd be easier to manage those select options if we ever wanted e.g. to add a new field to be returned etc.
But as I said - it's probably not for now, but it was a good moment to raise it.

Reply:
Did the structure of the response change? If yes then it should be a separate endpoint.

Reply:
Did the structure of the response change? If yes then it should be a separate endpoint.